README.md

```diff
@@ -61,12 +61,29 @@ print(my_opts[0].foo)
 
 k4FWCore is a CMake project. After setting up the dependencies (use for example `source /cvmfs/sw.hsf.org/key4hep/setup.sh`)
 
-
-```
-mkdir build install
-cd build;
+``` bash
+mkdir build
+cd build
 cmake ..
 make install
 ```
 
-
+## Implementing algorithms
+k4FWCore uses (or will use) `Gaudi::Functional` for executing algorithms. In
```

Comment:
I'd remove the "or will use". And then: "There are three different types of algorithms, depending on your use case:" And then the enumeration... if you open a ticket on me I can make the following text a bit more explicit - the complicated part is not in the 'operator' part, but in the templating. Which I would add.

Reply:
I have added a few more comments and a link to a more complete list that I found in the LHCb starterkit. If you want to change it I think you should be able to push to this branch